src/elements/content-sidebar/additional-tabs/AdditionalTabsLoading.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../common/_animations.scss';
+
+.bcs-additional-tab-loading-btn {
+    align-items: center;
+    background-color: transparent;
+    display: flex;
+    height: 60px;
+    justify-content: center;
+    pointer-events: none;
+    position: relative;
+    width: 59px; // 1px for left border
+}
+
+$fade-in-time: 0s;
+
+.bcs-additional-tab-loading-content {
+    @include loading-pulse(#f4f4f4, #eee, $fade-in-time);
```

Comment:
Do we have variables for these colors?

Reply:
eee has a color but f4f4f4 does not. Updated the eee's reference.